module/zfs/zap_micro.c

```diff
@@ -461,7 +461,7 @@ mzap_open(objset_t *os, uint64_t obj, dmu_buf_t *db)
 		 * 62 entries before we have to add 2KB B-tree core node.
 		 */
 		zfs_btree_create_custom(&zap->zap_m.zap_tree, mze_compare,
```

Comment:
I missed this during the first pass when I adapted kernel code to use the new b-tree search function. I will add this shortly.

Reply:
I just repushed to fix this.